crates/router/src/core/payments/operations.rs

```diff
@@ -91,6 +91,22 @@ pub trait GetTracker<F, D, R>: Send {
         mandate_type: Option<api::MandateTxnType>,
         merchant_account: &storage::MerchantAccount,
     ) -> RouterResult<(BoxedOperation<'a, F, R>, D, Option<CustomerDetails>)>;
+
+    #[allow(clippy::too_many_arguments)]
+    async fn modify_trackers<'a>(
+        &'a self,
+        _state: &'a AppState,
+        _payment_id: &api::PaymentIdType,
+        _request: &R,
+        _mandate_type: Option<api::MandateTxnType>,
+        _merchant_account: &storage::MerchantAccount,
+    ) -> RouterResult<(BoxedOperation<'a, F, R>, D, Option<CustomerDetails>)>
+    where
+        Self: Sized,
+    {
+        Err(report!(errors::ApiErrorResponse::InternalServerError))
+            .attach_printable("post connector update tracker not found")
+    }
```

Comment:
Is this block supposed to go in the current PR?

Reply:
No. These changes were pushed accidentally. Will revert these.